http.api/src/main/java/ru/tinkoff/qa/neptune/http/api/CommonBodyPublishers.java

```diff
@@ -125,4 +142,161 @@ private CommonBodyPublishers() {
     public static HttpRequest.BodyPublisher empty() {
         return noBody();
     }
+
+    /**
+     * Serializes given object to json string for the using it as a request body
+     *
+     * @param t is an object to be serialized to json string
+     * @param builder is used for the serialization
+     * @param charset of a resulted request body
+     * @param <T> is a type of the object to be serialized
+     * @return a BodyPublisher
+     */
+    public static  <T> HttpRequest.BodyPublisher jsonStringBody(T t, GsonBuilder builder, Charset charset) {
+        checkArgument(nonNull(builder), "Json builder should not be a null value");
+        checkArgument(nonNull(charset), "Char set should not be a null value");
+        return ofString(builder.create().toJson(t), charset);
+    }
+
+    /**
+     * Serializes given object to json string for the using it as a request body
+     *
+     * @param t is an object to be serialized to json string
+     * @param builder is used for the serialization
+     * @param <T> is a type of the object to be serialized
+     * @return a BodyPublisher
+     */
+    public static  <T> HttpRequest.BodyPublisher jsonStringBody(T t, GsonBuilder builder) {
+        return jsonStringBody(t, builder, UTF_8);
+    }
+
+    /**
+     * Serializes given object to json/xml string for the using it as a request body. The serializing is
```

Comment:
Simplify:
Serializes given object to json/xml string to be used as a request body.
(new line)
A serialization is performed by Jackson.